tests/framework/Optimizers/GeneticAlgorithms/discrete/unconstrained/testGAMaxwoRepTwoPointsCrossover.xml

```diff
@@ -0,0 +1,172 @@
+<?xml version="1.0" ?>
+<Simulation verbosity="debug">
+  <TestInfo>
+    <name>framework/Optimizers/GA.MaxwoReplacement</name>
+    <author>MohammadAbdo</author>
+    <created>2020-05-16</created>
+    <classesTested>GeneticAlgorithm</classesTested>
+    <description>
+      This test assesses the Genetic algorithm using the weighted sum found in myLocalSum.py function.
+      The nominal dimensionality of the test problem is 3.
+      The objective variable is ans. The problem in unconstrained, it is a maximization problem, and the sampling is from discrete variables without replacement.
+      The cross over mechanism used is the twoPointsCrossover algorithm
+    </description>
+    <analytic>
+      This test uses myLocalSum's analytic objective function.
+    </analytic>
+  </TestInfo>
+
+  <RunInfo>
+    <WorkingDir>MaxwoReplacementTwoPointsCrossover</WorkingDir>
+    <Sequence>optimize, print</Sequence>
+  </RunInfo>
+
+  <Steps>
+    <MultiRun name="optimize" re-seeding="2286">
+      <Input class="DataObjects" type="PointSet">placeholder</Input>
+      <Model class="Models" type="ExternalModel">myLocalSum</Model>
+      <Optimizer class="Optimizers" type="GeneticAlgorithm">GAopt</Optimizer>
+      <SolutionExport class="DataObjects" type="PointSet">opt_export</SolutionExport>
+      <Output class="DataObjects" type="PointSet">optOut</Output>
+      <Output class="OutStreams" type="Print">opt_export</Output>
```

Comment:
This line can be removed since you are using IOStep to Print the info.

Reply:
Fixed